### ISSUE TITLE
AuthHelperError error has happened! No access token was found!

### ISSUE THREAD

**Issue by lumenwrites** [supabase-community/auth-helpers#175]
Hi! When the user is not logged into my app, I'm getting an error:
```
AuthHelperError error has happened!
{"type":"cookie_not_found","message":"No access token was found!","source":"sb_auth_helpers"}
```

How can I disable it, or handle it correctly?

User not being logged in is a normal situation when someone is using my app, it shouldn't throw an error.

**Comment by silentworks** [comment]
That only shows up in during development. If you set your NODE_ENV to production that error should no longer be visible.

**Comment by lumenwrites** [comment]
Here's a screenshot of my logs on vercel:
![screencapture-vercel-lumenwrites-gdschool-CZ5GrFsXneg7ZoHLrJ9V7a9E1vRE-functions-2022-07-13-23_55_47](https://user-images.githubusercontent.com/4761235/178850867-e37ac886-6fe5-45cb-8205-52a0be8d77cb.png)

Also, to be honest, I'd rather not see it in development as well, it's bugging me. I mean, user not being logged in shouldn't lead to any kind of error in any environment, right? Because it's not an error, nothing is wrong with the app.

**Comment by lumenwrites** [comment]
@silentworks 
I just double checked, on Vercel the NODE_ENV environment is definitely set to production.

In my logs I see two messages, one after the other:

```
ERROR	{"type":"cookie_not_found","message":"No access token was found!","source":"sb_auth_helpers"}
INFO	NODE_ENV: production
```

The second message comes from my `console.log()`, making sure that `NODE_ENV` is set to the correct value.

**Comment by silentworks** [comment]
Fixed in release `@supabase/auth-helpers@0.6.8` https://github.com/supabase-community/auth-helpers/commit/8e0b747edcc3291fec06fb277491bae81b0fa0a1
